src/integration/pages/mostWatchedPage/afrique/__snapshots__/canonical.test.js.snap

```diff
@@ -150,18 +150,6 @@ Object {
 }
 `;
 
-exports[`Canonical Most Watched Page I can see the list items 1`] = `"0:16Vidéo, Test asset, Durée 0,1626 avril 2017"`;
```

Comment:
🤔

Reply:
Sorry that is horribly cryptic. Would we expect these to disappear?

Reply:
I know right?! 😄

Reply:
I ran int tests again and it wrote the snapshots back in. not sure what happened